packages/parser/lib/css_lexer.re

```diff
@@ -534,6 +552,16 @@ let rec get_next_token = buf => {
   | combinator => COMBINATOR(latin1(buf))
   | string => STRING(latin1(~skip=1, ~drop=1, buf))
   | important => IMPORTANT
+  | not_operator => NOT_OPERATOR(latin1(buf))
+  | (or_operator, whitespace) => OR_OPERATOR(latin1(buf))
+  | and_operator => AND_OPERATOR(latin1(buf))
+  | only_operator => ONLY_OPERATOR(latin1(buf))
+  | equal_sign => EQUAL_SIGN(latin1(buf))
+  | greater_than => GREATER_THAN(latin1(buf))
+  | less_than => LESS_THAN(latin1(buf))
+  | all_media_type => ALL_MEDIA_TYPE(latin1(buf))
+  | screen_media_type => SCREEN_MEDIA_TYPE(latin1(buf))
+  | print_media_type => PRINT_MEDIA_TYPE(latin1(buf))
```

Comment:
I'm not a fan of having all operators as different tokens, did you thought about unifying them?

Reply:
Based on https://github.com/davesnx/styled-ppx/pull/424#discussion_r1495995969 to individually generate tokens, but _some_ are unifiable